recipes-bsp/u-boot/files/ae350-ax45mp/0004-Enable-printing-OpenSBI-boot-log.patch

```diff
@@ -0,0 +1,27 @@
+From 421ebd09a6805b5e698d70d2c2ec9a63e144e068 Mon Sep 17 00:00:00 2001
+From: Yu Chien Peter Lin <peterlin@andestech.com>
+Date: Thu, 6 Jan 2022 13:50:07 +0800
+Subject: [PATCH] Enable printing OpenSBI boot log
```

Comment:
All of these patches should also go upstream, with a status message included in the commits: https://www.openembedded.org/wiki/Commit_Patch_Message_Guidelines#Importing_from_Elsewhere